BUILD

```diff
@@ -44,6 +43,15 @@ distribution_structure(
     visibility = ["//server:__pkg__", "//console:__pkg__"]
 )
 
+distribution_structure(
```

Comment:
`logback.xml` and `grakn.properties` should either be:
a) bundled as part of `grakn-core-server`
b) in its own target `grakn-core-conf`